.github/workflows/build_docs.yml

```
@@ -21,12 +20,9 @@ jobs:
       - name: Install common packages
         uses: ./.github/actions/install-dependencies
 
-      - name: Install book deps
-        run:
-          python3 -m pip install -r requirements.txt
-
       - name: Build the book
-        run: 
+        run: |
+          conda activate mpi
           jupyter-book build . -W
```

You can avoid activation troubles with `conda run`:

```suggestion
          conda run -n mpi jupyter-book build . -W
```